avivator/src/utils.js

```diff
@@ -298,3 +299,24 @@ export function guessRgb({ Pixels }) {
 
   return SamplesPerPixel === 3 || is3Channel8Bit || interleavedRgb;
 }
+export function truncateDecimalNumber(value, maxLength) {
+  if (!value && value !== 0) return '';
+  const stringValue = value.toString();
+  return stringValue.length > maxLength
+    ? stringValue.substring(0, maxLength).replace(/\.$/, '')
+    : stringValue;
+}
+
```

Comment:
why not https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed?

Reply:
I think the issue is its behavior on non-floats, like
```javascript
Number.parseFloat(1234).toFixed(6)
"1234.000000"
```

Reply:
Maybe there's another way of doing this?